### ISSUE TITLE
transformModulesToAmd + unbundled option fails to resolve bare imports

### ISSUE THREAD

**Issue by btopro** [Polymer/tools#3402]
the transformModulesToAmd option in build routinse in Polymer build does not correctly resolve bare dynamic imports. This is visible in firefox where it will convert it to a require statement but not actually alter the path, causing it to try and be relative to where ever the current file is.
```js
TypeError: "Failed to fetch https://cdn.waxam.io/build/es5-amd/node_modules/@lrnwebcomponents/haxcms-elements/lib/core/@lrnwebcomponents/haxcms-elements/lib/core/haxcms-editor-builder.js"
```
in this example: `https://cdn.waxam.io/build/es5-amd/node_modules/@lrnwebcomponents/haxcms-elements/lib/core/haxcms-site-builder.js` will call `import("@lrnwebcomponents/haxcms-elements/lib/core/haxcms-editor-builder.js")`. This works fine in ES6 but when it goes to path resolve in ES5 you get what you see above as opposed to any kind of `../../../../` being appended to make the path relative accurately

**Comment by btopro** [comment]
here's my polymer.json https://github.com/elmsln/HAXcms/blob/master/polymer.json#L47 which is leveraged to kick things over to that CDN.

**Comment by btopro** [comment]
TLDR;
- Good: transformModulesToAmd + bundled + dynamic imports
- Good: unbundled + dynamic imports
- Broken: transformModulesToAmd + unbundled + dynamic imports

This is an issue with unbundled AMD module compiling.
steps to reproduce:
- Test repo: https://github.com/btopro/lit-test
- run `polymer build`

This repo's output will WORK for the one marked unbroken and will break for the one marked broken. The only difference is the bundling. What's breaking appears to be a combination of transformModulesToAmd when using the unbundled flag. Dynamic imports Unbundled works fine without the AMD transform. AMD works fine without dynamic imports.

**Comment by btopro** [comment]
thanks to @Westbrook for making the repo to confirm I wasn't crazy

**Comment by usergenic** [comment]
Just published polymer-cli@1.9.9 and polymer-build@3.1.4 which incorporate your PR.

Please install and verify fix for your use case.  Reopen this issue if you have continued trouble.  :)

**Comment by btopro** [comment]
installed to double check and yes this appears to have fixed it. Thank you for accepting so quickly!
